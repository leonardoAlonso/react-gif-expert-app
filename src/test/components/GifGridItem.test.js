import React from 'react';
import { shallow } from 'enzyme';
import { GifGridItem } from '../../components/GifGridItem';
import '@testing-library/jest-dom';

describe('Debe mostrar correctamente el componente <GifGridItem>', () => {
    // let wrapper = shallow(<GifGridItem/>)

    // beforeEach(() =>{
    //     wrapper = shallow(<GifGridItem/>)
    // })

    const title = 'Soy un titulo';
    const url = 'https://localhost/algo/image.jpg';

    test('GifGridItem default', () => {
        const wrapper = shallow( <GifGridItem title={title} url={url}/> );
        expect(wrapper).toMatchSnapshot();
    })
})