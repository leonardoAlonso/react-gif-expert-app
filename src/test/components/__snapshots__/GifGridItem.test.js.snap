// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Debe mostrar correctamente el componente <GifGridItem> GifGridItem default 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="Soy un titulo"
    src="https://localhost/algo/image.jpg"
  />
  <p>
    Soy un titulo
  </p>
</div>
`;
